config-linux.md

```diff
@@ -530,24 +530,29 @@ If `intelRdt` is not set, the runtime MUST NOT manipulate any `resctrl` pseudo-f
 The following parameters can be specified for the container:
 
 * **`closID`** *(string, OPTIONAL)* - specifies the identity for RDT Class of Service (CLOS).
-    If `closID` is set, runtimes MUST create `closID` directory in a mounted `resctrl` pseudo-filesystem if it doesn't exist. If not set, runtimes MUST use the container ID from [`start`](runtime.md#start) and create the `<container-id>` directory.
 
 * **`l3CacheSchema`** *(string, OPTIONAL)* - specifies the schema for L3 cache id and capacity bitmask (CBM).
     The value SHOULD start with `L3:` and SHOULD NOT contain newlines.
 * **`memBwSchema`** *(string, OPTIONAL)* - specifies the schema of memory bandwidth per L3 cache id.
-    * The value MUST start with `MB:` and MUST NOT contain newlines.
+    The value MUST start with `MB:` and MUST NOT contain newlines.
 
-    * If both `l3CacheSchema` and `memBwSchema` are set, runtimes MUST write the combined value to the `schemata` file in that sub-directory discussed in `closID`.
+The following rules on parameters MUST be applied:
 
-    * If `l3CacheSchema` contains a line beginning with `MB:`, the value written to `schemata` file MUST be the non-`MB:` line(s) from `l3CacheSchema` and the line from `memBWSchema`.
+* If both `l3CacheSchema` and `memBwSchema` are set, runtimes MUST write the combined value to the `schemata` file in that sub-directory discussed in `closID`.
 
-    * If either `l3CacheSchema` or `memBwSchema` is set, runtimes MUST write the value to the `schemata` file in the that sub-directory discussed in `closID`.
+* If `l3CacheSchema` contains a line beginning with `MB:`, the value written to `schemata` file MUST be the non-`MB:` line(s) from `l3CacheSchema` and the line from `memBWSchema`.
```

Comment:
@marquiz I am looking at the specs to implement this in crun but I don't understand this rule.

Can you give an example?  Should it be dropped (at least I don't see any special handling in runc either for this)?

Reply:
@giuseppe the idea is that if you have 
```
l3CacheSchema: "L3:foobar\nMB:abcd"
memBWSchema: "MB:xyzy"
```

Then you'd actually write `L3:foobar\nMB:xyzy` to the schemata file.

I don't think this has been implemented anywhere. I didn't write this particular formulation, and TBH, I wouldn't probably bother implementing it. I think the fields and there handling is just ambiguously specified, e.g. somehow it's implied that `l3CacheSchema` could contain multiple lines but `memBWSchema` only one line. What if l3CacheSchema contains `L3:...` but  memBWSchema: contains `L3:...` (and other similar questions)...

I try to address the problems in #1230

Reply:
thanks, I understand now.  Still feels weird to specify that  `MB:` is filtered out instead of just not expecting such a case :-) but I guess we can live with it at this point